resources/assets/styles/components/block.scss

```diff
@@ -0,0 +1,11 @@
+[class^="wp-block-"] {
```

Comment:
drop this parent? global `.has-xxxxx-color` is okay

Reply:
d2bb13d